runtime/__init__.py

```diff
@@ -142,67 +149,59 @@ class Runtime:
     @staticmethod
     @functools.lru_cache(maxsize=1)
     def get() -> "Runtime":
-        """Gets a Runtime singleton.
+        """Gets the Runtime singleton.
 
         Raises:
-            ValueError: The requested config is not known.
-            ModuleNotFoundError: The prebuilt _portable_lib.so is not found.
+            ModuleNotFoundError: if the prebuilt _portable_lib.so is not found.
         """
-        try:
-            import executorch.extension.pybindings.portable_lib as legacy_module
-        except ModuleNotFoundError as e:
-            raise ModuleNotFoundError(
-                "Prebuilt <site-packages>/extension/pybindings/_portable_lib.so is not found. Please reinstall ExecuTorch from pip."
-            ) from e
+        import executorch.extension.pybindings.portable_lib as legacy_module
 
         return Runtime(legacy_module=legacy_module)
 
     def __init__(self, *, legacy_module: ModuleType) -> None:
-        # TODO: Expose the kernel callables to Python.
         # Public attributes.
-        self.operator_registry = OperatorRegistry()
+        self.operator_registry = OperatorRegistry(legacy_module)
         # Private attributes.
         self._legacy_module = legacy_module
 
     def load_program(
         self,
         data: Union[bytes, bytearray, BinaryIO, Path, str],
         *,
-        verification_config: Optional[Verification] = Verification.InternalConsistency,
+        verification: Verification = Verification.InternalConsistency,
     ) -> Program:
         """Loads an ExecuTorch program from a PTE binary.
 
         Args:
-            data: The binary program data to load; typically PTE data. Note that
-                this can also load PTE data that is wrapped inside a bundled
-                program, but it will not provide access to the bundled program's
-                test/validation data.
-            verification_config: The configuration for program verification.
+            data: The binary program data to load; typically PTE data.
+            verification: The configuration for program verification.
```

Comment:
"configuration" doesn't mean much here.
```suggestion
            verification: The level of program verification to perform.
```